README.markdown

```diff
@@ -153,6 +156,8 @@ If either `-s` or `-t` is not specified or has an empty list value, then no matc
 
 Existing response headers with the same name are always overridden. If you want to add headers incrementally, use the standard [add_header](http://nginx.org/en/docs/http/ngx_http_headers_module.html#add_header) directive instead.
 
+If the `-i` options is specified, then the headers will be added to the new values *only if* they doesnt exist.
```

Comment:
This line has English grammar errors:

"options" => "option"
"doesnt" => "do not"

Also "the headers will be added to the new values" do not make sense. The "to the new values" part can be removed.